clang/docs/ReleaseNotes.rst

```diff
@@ -88,6 +88,7 @@ sections with improvements to Clang's support for those languages.
 
 C++ Language Changes
 --------------------
+- Implemented _BitInt literal suffixes as ``__wb`` or ``__WB`` with unsigned modifiers also allowed. (#GH85223).
```

Comment:
I think we should mention that this is a Clang extension; also, don’t we have a separate section for Clang extensions?

Reply:
I see, I was misremembering then, my bad.